drivers/flash/flash_stm32_ospi.c

```diff
@@ -17,7 +17,7 @@
 #include <zephyr/drivers/clock_control/stm32_clock_control.h>
 #include <zephyr/drivers/clock_control.h>
 #include <zephyr/drivers/flash.h>
-#include <zephyr/dt-bindings/flash_controller/ospi.h>
+#include <zephyr/dt-bindings/flash_controller/xspi.h>
```

Comment:
You can't convert existing ospi driver into xspi driver since xspi HAL api is not available on H7 which uses this driver.
A full new file is required